### PR TITLE
Implement Gearman Message broker & PHP Extension

### PR DESCRIPTION
**Pull request by stefnats**
##### I completed the 3 steps below:

- [X] I've read the [Contribution Guide](http://laradock.io/contributing).
- [X] I've updated the **documentation**. (refer to [this](http://laradock.io/contributing/#update-the-documentation-site) for how to do so).
- [X] I enjoyed my time contributing and making developer's life easier :)

This is implements the Gearman message broker & its PHP Extension.

New .env Variables:

```env

WORKSPACE_INSTALL_GEARMAN
PHP_FPM_INSTALL_GEARMAN
PHP_WORKER_INSTALL_GEARMAN
GEARMAN_VERSION
GEARMAN_PORT
GEARMAN_VERBOSE
GEARMAN_QUEUE_TYPE
GEARMAN_THREADS
GEARMAN_BACKLOG
GEARMAN_FILE_DESCRIPTORS=
GEARMAN_JOB_RETRIES=0
GEARMAN_ROUND_ROBIN=0
GEARMAN_WORKER_WAKEUP=0
GEARMAN_KEEPALIVE=0
GEARMAN_KEEPALIVE_IDLE=30
GEARMAN_KEEPALIVE_INTERVAL=10
GEARMAN_KEEPALIVE_COUNT=5
GEARMAN_MYSQL_HOST=localhost
GEARMAN_MYSQL_PORT=3306
GEARMAN_MYSQL_USER=root
GEARMAN_MYSQL_PASSWORD=	
GEARMAN_MYSQL_PASSWORD_FILE=
GEARMAN_MYSQL_DB=Gearmand
GEARMAN_MYSQL_TABLE=gearman_queue
```

### CODE REVIEW

**Comment by Mahmoudz** [comment]
@stefnats thanks for the PR, if you have time please add gearman to the documentation under the list of supported `Message Brokers`.

**Comment by stefnats** [comment]
I believe i already did that, didn't i? Could you re-check?

File: DOCUMENTATION/content/introduction/index.md [hunk 1]
```diff
@@ -149,6 +149,7 @@ That's it! enjoy :)
     - Eclipse Mosquitto
     - PHP Worker
     - Laravel Horizon
+    - Gearman
```
Comment (stefnats) [inline DOCUMENTATION/content/introduction/index.md]: I added this note, or did you mean another one?

**Comment by stefnats** [comment]
@Mahmoudz just wanted to ask if you saw my note in the documentation or am i missing something?

**Comment by stefnats** [comment]
Just out of interest, why is this not merged?

**Comment by sohaibilyas** [comment]
@stefnats can you please look into https://github.com/laradock/laradock/issues/2436

in short, after setting up **PHP_WORKER_INSTALL_GEARMAN** to **true** its not installing gearman in **php-worker** container.

**Pull request merged**